en/nodes-custom.md

```diff
@@ -110,13 +128,33 @@ can both be overridden to execute the Curtin configuration in the ephemeral
 environment. Additionally, the `late` hook can be used to execute a
 configuration for a machine being installed, a state known as *in-target*.
 
+Curtin commands look like this:
```

Comment:
Note that this addresses a comment in the bug about not understanding how curtin commands work.